examples/arm/executor_runner/arm_executor_runner.cpp

```diff
@@ -45,10 +45,23 @@ char* model_pte = nullptr;
 #include "model_pte.h"
 #endif
 
-using namespace exec_aten;
```

Comment:
Note that `using namespace x` is forbidden by the ExecuTorch style guide (which is based on the Google style guide): https://google.github.io/styleguide/cppguide.html#Namespaces

Reply:
Just FYI, in PyTorch and internal codebase, `using namespace ` is indeed discouraged/prohibited in headers, but IMHO is totally fine in CPP files as it's limited to one compilation unit.

Reply:
There is the possibility that `using namespace`, even in a .cpp file, can cause an unnecessary build breakage. E.g., if we added a name to the `exec_aten` namespace that happened to conflict with a name defined in the .cpp file. Then, making a technically-FC/BC API change to the core headers could cause build breakages in the .cpp files that included them.

For now, ET mostly defers to the Google style guide. If we want to relax this "don't use `using namespace`" rule then we could do that if people wanted.